homeassistant/components/lektrico/config_flow.py

```diff
@@ -0,0 +1,166 @@
+"""Config flow for Lektrico Charging Station."""
+from __future__ import annotations
+
+from typing import Any
+
+import lektricowifi
+import voluptuous as vol
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import (
+    ATTR_HW_VERSION,
+    ATTR_SERIAL_NUMBER,
+    CONF_FRIENDLY_NAME,
+    CONF_HOST,
+    CONF_TYPE,
+)
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+
+class LektricoFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle a Lektrico config flow."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize flow."""
+        self._host: str
+        self._friendly_name: str
+        self._serial_number: str
+        self._board_revision: str
+        self._device_type: str
+
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
```

Comment:
All `FlowResult` should be changed to `ConfigFlowResult`

Reply:
I made the changes, thanks.